.travis.yml

```diff
@@ -4,15 +4,28 @@ branches:
     - master
 rvm:
   - 2.3
+  - 2.5
 gemfile:
   - gemfiles/rails_4_2_pundit_1.gemfile
   - gemfiles/rails_5_0_pundit_1.gemfile
   - gemfiles/rails_5_1_pundit_1.gemfile
   - gemfiles/rails_5_2_pundit_1.gemfile
+  - gemfiles/rails_6_0_pundit_1.gemfile
   - gemfiles/rails_4_2_pundit_2.gemfile
   - gemfiles/rails_5_0_pundit_2.gemfile
   - gemfiles/rails_5_1_pundit_2.gemfile
   - gemfiles/rails_5_2_pundit_2.gemfile
+  - gemfiles/rails_6_0_pundit_2.gemfile
+matrix:
+  exclude:
```

Comment:
This is the minimum number of exclusions since rails 4.2 doesn't like ruby 2.5 and rails 6 doesn't support ruby 2.3. Totally open to adding more exclusions so we're not running 16 builds

Reply:
Is it possible to only run Ruby 2.5 for Rails 6 runs, and keep 2.3 for other Rails versions?